### PR TITLE
fix(option): move default timeout to RequestOption

### PR DESCRIPTION
**Pull request by adambrunner**

### CODE REVIEW

File: wrapper.spec.js [hunk 1]
```diff
@@ -90,16 +91,18 @@ describe('Wrapper', function() {
...
 
-  it('should send GET request with given timeout on options', function*() {
-    var expectedRequestOption;
-    this.sandbox.stub(request, 'get', function(options, callback) {
-      expectedRequestOption = options;
-      callback(null, apiResponse);
-    });
+  describe('timeout', function() {
+    it('should send GET request with given timeout in options', function*() {
+      var expectedRequestOption;
```
Comment (gabor-balla-emarsys) [inline wrapper.spec.js]: I think this should be the actualRequestOptions

**Comment by drawain** [comment]
It looks like a refactoring instead of fix by the view of usage

**Comment by adambrunner** [comment]
@drawain yes, but semantic release does not create a new version if `refactor()` is the type

**Comment by drawain** [comment]
Ok, otherwise looks good

**Pull request merged**